### PR TITLE
feat(Emby): add support for new and old versions

### LINKED ISSUE #8749: Emby |  * - Latest version breaks presence, only browsing
### Steps to Reproduce

With Emby v4.8.8 it seems the rich presence assumes I am always browsing music or (in 4.9 beta) at the home page, even if I play music.

### Expected Behavior

I expected the emby presence to show the music I am currently playing

### Actual Behavior

The emby presence says I am on the home page, or browsing. It never shows the music or content I am playing.

### Page URL

https://us-e-argon.firehostredux.com:2096/

### Screenshots

image

### Operating System

Windows 11

### Browser and Version

Firefox 130.0.1 (64-bit)

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by darkvillager2**
## Description 

Resolves #8749
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/E/Emby/presence.ts [hunk 1]
```diff
@@ -460,24 +513,39 @@ const mediaInfoCache = new Map<string, MediaInfo>();
...
-
-	let { AccessToken: accessToken } = ApiClient._serverInfo;
+	let ae, accessToken: string;
+	if (!version.includes("4.7")) {
+		ae = ApiClient._serverInfo as ServerLatest;
+		accessToken = ae?.Users?.[1]?.AccessToken;
+	} else {
+		ae = ApiClient._serverInfo as Server;
+		accessToken = ae?.AccessToken;
+	}
```
Comment (darkvillager2) [inline websites/E/Emby/presence.ts:523]: If there's a better solution let me know

File: websites/E/Emby/presence.ts [hunk 2]
```diff
@@ -460,24 +513,39 @@ const mediaInfoCache = new Map<string, MediaInfo>();
...
+			/\//gm,
+			""
+		)}?` +
+		`X-Emby-Client=${ApiClient._appName.replace(/ /gm, "_")}&` +
+		`X-Emby-Device-Name=${ApiClient._deviceName}&` +
+		`X-Emby-Device-Id=${ApiClient._deviceId}&` +
+		`X-Emby-Client-Version=${ApiClient._appVersion}&` +
+		`X-Emby-Token=${accessToken}`;
+
+	console.log(url.replace(/ /gm, "_"));
```
Comment (darkvillager2) [inline websites/E/Emby/presence.ts:547]:
```suggestion
```

File: websites/E/Emby/presence.ts (hunk 1 above)
Comment (Bas950) [reply websites/E/Emby/presence.ts:523]: Maybe add the semver package and use one of their functions if it's a small dependency, on phone can't check rn xd

Comment (darkvillager2) [reply websites/E/Emby/presence.ts:523]: > Also this currently matches on x.4.2 and 4.2.0 idk which you even want

????

Comment (Bas950) [reply websites/E/Emby/presence.ts:523]: > > Also this currently matches on x.4.2 and 4.2.0 idk which you even want
> 
> ????

I meant 7 instead of 2..

Comment (darkvillager2) [reply websites/E/Emby/presence.ts:523]: Looking into it

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**